content/en/docs/concepts/configuration/even-pods-spread.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Even Pods Spread
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page explains how EvenPodsSpread feature works and the mechanics under the hood.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## EvenPodsSpread
+
+`EvenPodsSpread` feature gives users more fine-grained control on the pods' distribution across the cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
+
+## Motivation
+
+In Kubernetes, "Affinity" related directives are aimed to control how pods are
+scheduled - more packed or more scattered. But right now only limited options
+are offered: for `PodAffinity`, infinite pods can be packed into qualifying
+topology domain(s); for `PodAntiAffinity`, only one pod can be scheduled into a
+single topology domain.
+
+This is not an ideal situation if users want to distribute pods evenly across different
+topology domains - to achieve high availability or cost-saving. And
+regular rolling upgrade or scaling out replicas can also be problematic.
+
+## Set up
+
+### Enable Feature Gate
+
+Ensure the `EvenPodsSpread` feature gate is enabled (it is disabled by default
+in 1.16). See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) for an explanation of enabling feature gates. The `EvenPodsSpread` feature gate must be enabled on apiserver **and** scheduler.
+
+### Apply Labels to Worker Nodes
+
+This feature requires the nodes to be labelled with topological key-value pairs via `kubectl label node <node_name> <key1=val1> <key2=val2> ...`. For example, the following 4-node cluster is tagged with {node: node_name>, zone: zone_name}:
+
+```
+$ kubectl get node --show-labels
```

Comment:
Per the style guide:
- avoid putting the prompt into shell snippets
- separate the command from its sample output